### ISSUE TITLE
Error while generating PDF

### ISSUE THREAD

**Issue by TomLorenzi** [jonaslejon/malicious-pdf#4]
![image](https://user-images.githubusercontent.com/57677046/130259969-9214cac0-85a0-46f4-93f5-7497536dced9.png)

Using python 3.9

**Comment by kayshinonome** [comment]
Same, on python 3.9.6
```
Creating PDF files..
Traceback (most recent call last):
  File "/home/kayshinonome/Code/malicious-pdf/./malicious-pdf.py", line 613, in <module>
    create_malpdf("test1.pdf", '\\\\' + '\\\\'  + host + '\\\\' )
  File "/home/kayshinonome/Code/malicious-pdf/./malicious-pdf.py", line 533, in create_malpdf
    file.write('''
TypeError: a bytes-like object is required, not 'str'
```

**Comment by kayshinonome** [comment]
Python 3.9.6
```
Creating PDF files..
Traceback (most recent call last):
  File "/home/kayshinonome/Code/malicious-pdf/./malicious-pdf.py", line 613, in <module>
    create_malpdf("test1.pdf", '\\\\' + '\\\\'  + host + '\\\\' )
  File "/home/kayshinonome/Code/malicious-pdf/./malicious-pdf.py", line 533, in create_malpdf
    file.write('''
TypeError: a bytes-like object is required, not 'str'
```

**Comment by x1mdev** [comment]
```
Creating PDF files..
Traceback (most recent call last):
  File "/Users/x1m/bugbounty/envicio/malicious-pdf/malicious-pdf.py", line 613, in <module>
    create_malpdf("test1.pdf", '\\\\' + '\\\\'  + host + '\\\\' )
  File "/Users/x1m/bugbounty/envicio/malicious-pdf/malicious-pdf.py", line 533, in create_malpdf
    file.write('''
TypeError: a bytes-like object is required, not 'str'
```

Python 3.9.6

**Comment by x1mdev** [comment]
```
Creating PDF files..
Traceback (most recent call last):
  File "/Users/x1m/malicious-pdf/malicious-pdf.py", line 613, in <module>
    create_malpdf("test1.pdf", '\\\\' + '\\\\'  + host + '\\\\' )
  File "/Users/x1m/malicious-pdf/malicious-pdf.py", line 533, in create_malpdf
    file.write('''
TypeError: a bytes-like object is required, not 'str'
```

Python 3.9.6

**Comment by inhibitor181** [comment]
Also broken in Python 3.8.6.

But it worked via `python malicious-pdf.py` (Python 2.7.18)

**Comment by mclancy10006** [comment]
I think the io.FileIO needs to be changed to just open() for python3

**Comment by jonaslejon** [comment]
Thanks. Fixed in 5cad6e7e53d468e6b04b86c9387cf6a95045ed9f
